hub/powertoys/cmd-not-found.md

```diff
@@ -0,0 +1,37 @@
+---
+title: PowerToys Command Not Found for Windows
+description: A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+ms.date: 02/01/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+
+![Command Not Found screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
+ - [PowerShell 7](/PowerShell/scripting/install/installing-PowerShell-on-windows)
+ - [PowerShell Microsoft.WinGet.Client module](https://www.powershellgallery.com/packages/Microsoft.WinGet.Client)
+
+## Install the module
+
+To install the Command Not Found module, navigate to the Command Not Found page in PowerToys settings and select the **Install** button. Once the module installation has completed, PowerShell 7 experimental features needed for the module to function will be enabled:
+
+ - PSFeedbackProvider
+ - PSCommandNotFoundSuggestion
+
+After that, PowerShell profile file will be appended with following block of PowerShell commands:
+```
```

Comment:
What's the language for this code snippet? psh?

Reply:
yes

Reply:
```suggestion

```psh
```